index.html

```diff
@@ -1,4 +1,37 @@
 <!DOCTYPE html>
-<html>
-  
+<html lang="es">
+    <head>
+        <title>Index Black Music</title>
+        <link rel="icon" href="img/simpleLogo.png">
+    </head>  
+    <body>
+        <h1>
+            BACK MUSIC
+        </h1>
+        <nav>
+            <a href="index.html">Index</a>
+            <a href="principal.html">P&aacute;gina principal</a>
+            <a href="SeccionCompras/compras.html">Secci&oacute;n de Compra</a>
+            <a href="SeccionVentas/ventas.html">Secci&oacute;n de Venta</a>
+            <a href="Valoraciones/valoraciones.html">Valoraciones</a>
+            <a href="contacto.html">Contacto</a>
+            <a href="detalles.html">Detalles</a>
+            <a href="miembros.html">Miembros</a>
+            <a href="bocetos.html">Bocetos</a>
+            <a href="planificacion.html">Planificaci&oacute;n</a>
+        </nav>
+        <section>
+            <h2>
+                &iquest;Qu&eacute; es Black Music?
```

Comment:
Quitad las entidades y se verá bien cuando tengáis la cabecera <meta charset ...>